### DISCUSSION TITLE
Is it possible to use more colors then primary,secondary,warning,success and danger?

### DISCUSSION THREAD

**Discussion by lucacastelnuovo** [filamentphp/filament#6377] [Help] [answered]
https://filamentphp.com/docs/2.x/tables/columns/badge

```php
use Filament\Tables\Columns\BadgeColumn;
 
BadgeColumn::make('status')
    ->colors([
        'primary',
        'secondary' => static fn ($state): bool => $state === 'draft',
        'warning' => static fn ($state): bool => $state === 'reviewing',
        'success' => static fn ($state): bool => $state === 'published',
        'danger' => static fn ($state): bool => $state === 'rejected',
    ])
```

**Comment by danharrin** [comment]
You could use a ViewColumn with whatever color you want

**Reply by lucacastelnuovo** [reply]
Sure, if possible it would be awesome to pass custom classes, tailwind base color like (blue) or hex values.

**Reply by danharrin** [reply]
You could PR that in if you want, using Tailwind classes

**Reply by lucacastelnuovo** [reply]
I saw that info colors are already being added in v3. My needs are met but if you think it is still worth it for others I could try to make a PR for it.

**Comment by lucacastelnuovo** [comment, marked as answer]
I looked into the code, and you can already pass tailwind classes to use a custom colour (or any other class).
Just make sure that the tailwind classes are compiled

```php
	// ...
	->color('text-blue-700 bg-blue-500/10')
```

**Reply by michaelklopf** [reply]
Hey, what do you mean with making sure to compile the tailwind classes? The "normal" build process is running but the color is not set. Viewing the css classes in the browser it always falls back to `text-custom-600`.

**Reply by lucacastelnuovo** [reply]
Hi,

What I meant was you should make sure the PHP in which you return a custom class string should be inside the tailwind.config.js content array.
